tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-boleto-test.php

```
@@ -0,0 +1,48 @@
+
+		$initial_account = $wc_stripe->account;
+		try {
+			$wc_stripe->account = $mock_account;
+
+			$payment_method = new WC_Stripe_UPE_Payment_Method_Boleto();
+
+			$this->assertSame( $expected_result, $payment_method->is_available_for_account_country() );
+		} finally {
+			$wc_stripe->account = $initial_account;
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_is_available_for_account_country is not named in camelCase. (undefined)

(CamelCaseMethodName)